CefSharp/IContextMenuParams.cs

```diff
@@ -125,7 +125,7 @@ public interface IContextMenuParams : IDisposable
         /// Returns true if the context menu was invoked from a pepper plugin.
         /// </summary>
         bool IsPepperMenu { get; }
-
+        string TitleText { get; }
```

Comment:
Could you add a small summary explaining what the property returns (the "title" attribute of img element)?
Including it "should" handle the "alt" attribute but it doesn't

Reply:
Done